[2022-06-25 19:41:48,818] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: CSV_TO_DS.SEND_DS_md_currency_d scheduled__2022-06-22T21:26:59.328971+00:00 [queued]>
[2022-06-25 19:41:48,872] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: CSV_TO_DS.SEND_DS_md_currency_d scheduled__2022-06-22T21:26:59.328971+00:00 [queued]>
[2022-06-25 19:41:48,872] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-25 19:41:48,872] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-25 19:41:48,872] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-25 19:41:48,895] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): SEND_DS_md_currency_d> on 2022-06-22 21:26:59.328971+00:00
[2022-06-25 19:41:48,911] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'CSV_TO_DS', 'SEND_DS_md_currency_d', 'scheduled__2022-06-22T21:26:59.328971+00:00', '--job-id', '1297', '--raw', '--subdir', 'DAGS_FOLDER/csv_to_ds.py', '--cfg-path', '/tmp/tmpzm0wmfln', '--error-file', '/tmp/tmp5eowkt1g']
[2022-06-25 19:41:48,912] {standard_task_runner.py:77} INFO - Job 1297: Subtask SEND_DS_md_currency_d
[2022-06-25 19:41:48,900] {standard_task_runner.py:52} INFO - Started process 231 to run task
[2022-06-25 19:41:49,109] {logging_mixin.py:109} INFO - Running <TaskInstance: CSV_TO_DS.SEND_DS_md_currency_d scheduled__2022-06-22T21:26:59.328971+00:00 [running]> on host 1a36ba6d0ffa
[2022-06-25 19:41:49,297] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=CSV_TO_DS
AIRFLOW_CTX_TASK_ID=SEND_DS_md_currency_d
AIRFLOW_CTX_EXECUTION_DATE=2022-06-22T21:26:59.328971+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-22T21:26:59.328971+00:00
[2022-06-25 19:41:49,461] {FromLocal.py:71} INFO - Table md_currency_d read in pandas success. 
[2022-06-25 19:41:49,514] {FromLocal.py:72} INFO - 
    currency_rk data_actual_date  ... currency_code code_iso_char
0      4586704       2011-09-06  ...           000           NON
1           50       2017-05-11  ...           356           INR
2           51       2017-05-11  ...           484           MXN
3           52       2017-05-11  ...           434           LYD
4           53       2017-05-11  ...           422           LBR

[5 rows x 5 columns]
[2022-06-25 19:41:49,646] {FromLocal.py:96} INFO - (psycopg2.errors.DatatypeMismatch) column "data_actual_date" is of type date but expression is of type text
LINE 2:     SELECT DISTINCT ON (CURRENCY_RK, DATA_ACTUAL_DATE) *
                                                               ^
HINT:  You will need to rewrite or cast the expression.

[SQL: INSERT INTO ds.md_currency_d
    SELECT DISTINCT ON (CURRENCY_RK, DATA_ACTUAL_DATE) *
    FROM temp.md_currency_d        
ON CONFLICT (CURRENCY_RK, DATA_ACTUAL_DATE)
DO UPDATE 
	SET 
		DATA_ACTUAL_END_DATE = excluded.DATA_ACTUAL_END_DATE,
		CURRENCY_CODE = excluded.CURRENCY_CODE,
		CODE_ISO_CHAR = excluded.CODE_ISO_CHAR;]
(Background on this error at: http://sqlalche.me/e/13/f405)
[2022-06-25 19:41:54,658] {python.py:175} INFO - Done. Returned value was: None
[2022-06-25 19:41:54,727] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=CSV_TO_DS, task_id=SEND_DS_md_currency_d, execution_date=20220622T212659, start_date=20220625T194148, end_date=20220625T194154
[2022-06-25 19:41:55,295] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-25 19:41:55,444] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
